sokutei.h

```diff
@@ -0,0 +1,72 @@
+#ifndef SOKUTEI_BENCHMARK
+#define SOKUTEI_BENCHMARK
+
+#define MAX_COUNTERS 100
+#define COUNTER_LENGTH 32
+#define INT int
+
+#ifdef SOKUTEI_NO_STDLIB
+#ifndef SOKUTEI_STDLIB
+#define SOKUTEI_STDLIB
+
+int sokutei_strcmp(const char *a, const char *b){
+    int i = 0;
+    while(a[i] && b[i] && a[i] == b[i]) i++;
+    return a[i] - b[i];
+}
+#define STRCMP(a, b) sokutei_strcmp(a, b)
+
+char *sokutei_strcpy(char *a, const char *b){
```

Comment:
I know, that C code generally is concise and hard to read. But it does not have to be that way! What do you think, are these parameter names `a` and `b` are readable? In my opinion, not quite.